src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
```

Comment:
The variable name 'bothside' is not descriptive enough. It's unclear what 'bothside' means in the context of a right triangle. Consider renaming it to something more descriptive, such as 'base' or 'hypotenuse', depending on which side it represents.